### PR TITLE
Create SPIDER_QUEENS_STINGER.json

### PR DESCRIPTION
**Pull request by ZviWolf**
Added Spider Queen's Stinger (1st time doing this so might be fked up)

### CODE REVIEW

File: items/SPIDER_QUEENS_STINGER.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "B1": "ENCHANTED_SPIDER_EYE:6",
+    "B2": "",
+    "B3": "SOUL_STRING:6",
+    "C1": "",
+    "C2": "ENCHANTED_DARK_OAK_LOG:6",
+    "C3": "SOUL_STRING:6"
+  },
+  "internalname": "SPIDER_QUEENS_STINGER",
+  "clickcommand": "viewrecipe",
+  "modver": "1.0.0",
```
Comment (DeDiamondPro) [inline items/SPIDER_QUEENS_STINGER.json:31]: Mod version should be 2.0.0-REL

File: items/SPIDER_QUEENS_STINGER.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "itemid": "minecraft:bow",
+  "displayname": "§5Spider Queen's Stinger",
+  "nbttag": "{Unbreakable:1,HideFlags:254,display:{Lore:[0:\"§7Damage: §c+180\",2:\"§7Arrows shot using this bow have\",3:\"§7an aura around them that deals\",4:\"§c835.2❁ Damage §7to nearby\",5:\"§7enemies. Arrows travel through\",6:\"§7enemies.\",7:\"\",8:\"§8This item can be reforged!\",9:\"§5§lEPIC BOW\"],Name:\"§5Spider Queen's Stinger\"},ExtraAttributes:{id:\"SPIDER_QUEENS_STINGER"}}",
```
Comment (DeDiamondPro) [inline items/SPIDER_QUEENS_STINGER.json:4]: You forgot to escape the “ at the very end before the double } escape that ( add ) and you should be good

**Review by DeDiamondPro — CHANGES_REQUESTED** [review]
You just need to do some small changes and then you will be fine

**Comment by IRONM00N** [comment]
I fixed it for you.

**Pull request merged**